lms/templates/static_templates/server-error.html

```diff
@@ -1,16 +1,17 @@
+<%namespace name='static' file='../static_content.html'/>
 <%! from django.utils.translation import ugettext as _ %>
 <%inherit file="../main.html" />
 
 <section class="outside-app">
   <h1>
     ${_(u"There has been a 500 error on the {platform_name} servers").format(
-        platform_name=u"<em>{platform_name}</em>".format(platform_name=settings.PLATFORM_NAME)
+        platform_name=u"<em>{platform_name}</em>".format(platform_name=static.get_platform_name())
     )}
   </h1>
   <p>
     ${_(u'Please wait a few seconds and then reload the page. If the problem persists, please email us at {email}.').format(
         email=u'<a href="mailto:{email}">{email}</a>'.format(
-            email=settings.TECH_SUPPORT_EMAIL
+            email=static.get_value('email_from_address', settings.TECH_SUPPORT_EMAIL)
```

Comment:
Right, what I am requesting is to replace the direct reference to this particular setting in the template with an abstraction.  This will introduce some flexibility.